README.adoc

```diff
@@ -192,8 +188,8 @@ the database table and is not associated with an EntityManger. Managed entities
 been persisted, or fetched by `find()` or a query, and are managed by the EntityManager. This means that 
 they are actively tracked by the persistence context. Detached entities are no longer managed by a 
 persistence context. When using a container-managed EntityManager, entities automatically become 
-detached at the end of transaction boundaries. Lastly, removed entities are ones that have been removed 
-from the database. 
+detached at the end of transaction boundaries. Lastly, removed entities will be marked for removal 
+and will be removed from the database upon transaction commit. 
```

Comment:
and will be deleted from the database -- just to clarify an entity being marked for removal, and a table row being DELETED when JPA pushes the sql to the database.